WcaOnRails/db/migrate/20170517213035_update_list_of_countries.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+class UpdateListOfCountries < ActiveRecord::Migration[5.0]
+  def up
+    Country.delete_all
+    Country::ALL_STATES.each do |state|
+      Country.create(state)
```

Comment:
`create` -> `create!`? We want to know if validations prevent this country from getting created, right?